airflow/www/static/css/main.css

```diff
@@ -288,10 +288,20 @@ label[for="timezone-other"],
   font-weight: normal;
 }
 
+#timezone-menu {
+  overflow-x: hidden;
+}
+
 #timezone-menu ul.typeahead.dropdown-menu {
   max-height: 200px;
   overflow-y: auto;
   overflow-x: hidden;
+  border: none;
```

Comment:
We need the border css for our other autocomplete menus like searching for a DAG.

Reply:
Do you want me to remove "border: none" ??
and add some border??

Reply:
@bbovenzi 
After removing "border: none"
![image](https://github.com/user-attachments/assets/4447bec3-19ab-445b-981a-b3f01ce442ba)

Reply:
Sorry, what I meant is that we need to add the css change I suggested earlier:

<img width="265" alt="Screenshot 2024-07-31 at 1 28 29 PM" src="https://github.com/user-attachments/assets/a5282e6f-9e01-4512-a6dd-3b921ba64029">
<img width="420" alt="Screenshot 2024-07-31 at 1 28 35 PM" src="https://github.com/user-attachments/assets/dc36cfbe-121d-46f3-85b3-8c58c3aa45eb">

What browser are you using that you don't see the blue border?

Reply:
@bbovenzi 
I already added those css changes, that's why i'm not getting the blue border

Reply:
And also updated the PR.

Reply:
@bbovenzi any updates??

Reply:
Are you sure you're committing these css to the right branch?I haven't seen any css changes since I've commented on this PR. I pulled down your branch and ran it locally. The blue border still persists.

Reply:
@bbovenzi Can you check now